Conch/lib/Conch/Controller/Orc/Lifecycles.pm

```diff
@@ -62,6 +62,30 @@ sub create ($c) {
 	$c->status(303 => "/o/lifecycle/".$l->id);
 }
 
+=head2 update
+
+Update an existing lifecycle
+
+=cut
+
+sub update ($c) {
```

Comment:
We should be checking a role ID in the payload is valid and exists in the database if present.

Reply:
Also need to check if the role ID is not already taken by another lifecycle, since it's unique.